app/rvo.py

```diff
@@ -162,10 +163,8 @@ async def logged_out() -> Template:
 
 @get(path="/test", guards=[rvo_auth.authenticated_guard], include_in_schema=False)
 async def list_users(db_session: AsyncSession) -> Template:
-    users = await get_user_list(
-        db_session,
-        options=selectinload(User.notifications),
-    )
+    users_service: UserService = UserService(session=db_session, load=[User.notifications])
```

Comment:
Nit: would it make sense to have a `provide_users_with_notifications_service`, the same way we have a `provide_users_with_registrations_service`?

Reply:
see https://github.com/numerique-gouv/ami-notifications-api/pull/206#discussion_r2490674022